frame/evm/src/lib.rs

```diff
@@ -54,9 +54,12 @@
 #![cfg_attr(not(feature = "std"), no_std)]
 
 #[cfg(any(test, feature = "runtime-benchmarks"))]
-pub mod benchmarks;
-#[cfg(any(test, feature = "runtime-benchmarks"))]
-mod mock;
+pub mod benchmarking;
+
+// #[cfg(any(test, feature = "runtime-benchmarks"))]
+// pub mod benchmarks;
+// #[cfg(any(test, feature = "runtime-benchmarks"))]
+// mod mock;
```

Comment:
is `mock` not needed? does this break something else (tests)?

Reply:
Yeah I missed that and readded, it needed to be enabled only with `#[cfg(test])` since it will break during `runtime-benchmarks`.